### ISSUE TITLE
Not displaying snake_case_method names

### ISSUE THREAD

**Issue by antonkomarev** [sempro/phpunit-pretty-print#2]
I don't know why, but I don't see methods names in output.

![screenshot from 2017-12-23 23-47-39](https://user-images.githubusercontent.com/1849174/34322579-b6e9f1ba-e83b-11e7-9c2a-ee3f8fb7d277.png)

Methods looks like this:
```php
/** @test */
public function authenticated_user_cannot_authenticate()
{
    // Some assertions
}

**Comment by antonkomarev** [comment]
It seems to be related to #1

**Comment by ecrmnn** [comment]
Closed with #1
